packages/mobile/src/send/TransferConfirmationCard.tsx

```diff
@@ -67,56 +60,47 @@ const renderTopSection = (props: Props) => {
 }
 
 const renderAmountSection = (props: Props) => {
-  const { currency, type, value } = props
-
-  // tslint:disable react-hooks-nesting
-  const localCurrencyCode = useLocalCurrencyCode()
-  const localCurrencySymbol = useLocalCurrencySymbol()
-  const localValue = useDollarsToLocalAmount(value) || 0
-  // tslint:enable react-hooks-nesting
-  const transactionValue = getMoneyDisplayValue(
-    currency === CURRENCY_ENUM.DOLLAR && localCurrencyCode ? localValue : value
-  )
+  const { amount, type } = props
 
   switch (type) {
-    case TransactionTypes.INVITE_SENT: // fallthrough
-    case TransactionTypes.INVITE_RECEIVED:
+    case TransactionType.InviteSent: // fallthrough
+    case TransactionType.InviteReceived:
       return null
-    case TransactionTypes.NETWORK_FEE:
+    case TransactionType.NetworkFee:
       return (
-        <MoneyAmount
-          symbol={CURRENCIES[currency].symbol}
-          amount={getNetworkFeeDisplayValue(value, true)}
+        <CurrencyDisplay
+          type={DisplayType.Big}
+          amount={amount}
+          // tslint:disable-next-line: jsx-no-lambda
+          formatAmount={(value) => getNetworkFeeDisplayValue(value, true)}
```

Comment:
Is there particular reason to use lambda function?

Reply:
No, let me change this 👍